### PR TITLE
Krzysiek.update calendar

### LINKED ISSUE #156: The package is missing update calendar method
JS part is already prepared. We just need to create R functions.

### PR DESCRIPTION
**Pull request by krzysiek-frappsilon**
# Method for updating calendar 
closes #156

### CODE REVIEW

File: inst/www/shiny-semantic-calendar.js [hunk 1]
```diff
@@ -90,20 +90,31 @@ $.extend(semanticDateBinding, {
   },
 
   receiveMessage: function(el, data) {
+
+    console.log(data);
```
Comment (dokato) [inline inst/www/shiny-semantic-calendar.js]: to be removed

File: examples/app.R [hunk 2]
```diff
@@ -288,9 +288,20 @@ sidebar <- function() {
...
-              a(class="item", href="#tabset", "Tabset")
+              a(class="item", href="#tabset", "Tabset"),
+              a(class="item", href="#calendar", "Calendar")
               )))
 }
+
+calendar <- function() {
+  div(
+    h1(class="ui header", id="calendar", "Calendar"),
+    demo(uicalendar("date", type = "date", value = "20.2.2020", placeholder = "Select Date", min = "2.2.2020", max = "25.2.2020")),
```
Comment (dokato) [inline examples/app.R]: did that pass linter? can you break the line?

Comment (krzysiek-frappsilon) [reply examples/app.R]: My mistake, I didn't run it. It had length 132 so wouldn't pass.
Fixed it + fixed small style issues in calendar.R

File: inst/www/shiny-semantic-calendar.js [hunk 3]
```diff
@@ -92,18 +92,25 @@ $.extend(semanticDateBinding, {
...
+      this.setValue(el, data.value);
+      $(el).attr("data-date", this.getValue(el));
 
     $(el).trigger('change');
   }
 });
 
 Shiny.inputBindings.register(semanticDateBinding, 'shiny.semanticDate');
+
+Shiny.addCustomMessageHandler('updateSemanticCalendar', function(message){
```
Comment (galachad) [inline inst/www/shiny-semantic-calendar.js]: This is not used anywhere and it's probably not needed.

**Pull request merged**